src/components/UI/Text/Textarea.vue

```diff
@@ -220,5 +233,32 @@ export default {
     },
   },
   emits: ["input", "keypress", "keydown", "update:value", "start-icon-selected"],
+  // quilljs editor
+  mounted() {
+    this.editor = new Quill(this.$refs.editor, {
+      modules: {
+        toolbar: [["bold", "italic", "underline"]],
+      },
+      theme: "snow",
+      formats: ["bold", "underline", "italic"],
+    });
+
+    this.editor.root.innerHTML = this.value;
```

Comment:
If you're using value here then I guess you don't the variable `localValue` anymore right?

Reply:
No, I don't require that. Also, I was about to question why `localValue` was included in the code. Should I remove its instances from the code because we can do the functions with only the `value` variable?